apps/website/lib/src/routes/index.dart

```diff
@@ -16,56 +18,83 @@ HTMLElement get index => basicLayout(
     ..style.flexDirection = 'column'
     ..style.alignItems = 'center'
     ..style.gap = '2rem'
-    ..appendAll([_mainContent..style.minHeight = '70vh', _schedule]),
+    ..appendAll([
+      _mainContent,
+      sectionLayout('Schedule', _schedule)..style.marginTop = '2rem',
+    ]),
 );
 
-HTMLElement get _mainContent =>
-    HTMLElement.article()
-      ..style.display = 'flex'
-      ..style.flexDirection = 'column'
-      ..style.justifyContent = 'center'
-      ..style.alignItems = 'center'
-      ..appendAll([
-        HTMLHeadingElement.h1()..append(
-          HTMLImageElement()
-            ..style.width = '9rem'
-            ..alt = '${site.title} logo'
-            ..src = '/img/icon_flutterkaigi.svg',
-        ),
-        HTMLParagraphElement()
-          ..style.fontSize = '2rem'
-          ..textContent = formatDate(event.date),
-        HTMLDListElement()
-          ..style.display = 'grid'
-          ..style.gridGap = '0.5rem'
-          ..style.gridTemplateColumns = 'repeat(2, 1fr)'
-          ..style.marginTop = '1rem'
-          ..style.fontSize = '1.2rem'
-          ..appendAll([
-            HTMLElement.dt()..textContent = text(contents.placeLabel),
-            HTMLElement.dd()..append(
-              externalLink(text(event.place.name), url: event.place.url)
-                ..style.fontSize = 'inherit',
-            ),
-          ]),
-        HTMLParagraphElement()
-          ..style.position = 'relative'
-          ..style.marginTop = '1rem'
-          ..appendAll([
-            externalLink(
-              text(event.blog.sponsorship.title),
-              url: text(event.blog.sponsorship.url),
-            ),
-            HTMLImageElement()
-              ..style.position = 'absolute'
-              ..style.left = '100%'
-              ..style.bottom = '-50%'
-              ..style.height = '5rem'
-              ..alt = 'FlutterKaigi Dash'
-              ..src = '/img/flutterkaigi_dash.png',
-          ]),
-        _countdown..style.marginTop = '1rem',
-      ]);
+HTMLElement get _mainContent {
+  final tagLineWords = event.tagLine.split(' ');
+
+  return HTMLElement.article()
+    ..style.display = 'flex'
+    ..style.flexDirection = 'column'
+    ..style.justifyContent = 'center'
+    ..style.alignItems = 'center'
+    ..appendAll([
+      HTMLDivElement()
```

Comment:
ここ，ヘッダーで囲って中をspanにした方がいい気がします
```suggestion
      HTMLHeadingElement.h1()
```